packages/dropdowns/package.json

```diff
@@ -51,6 +50,6 @@
     "access": "public"
   },
   "zendeskgarden:library": "GardenDropdowns",
-  "zendeskgarden:max_size": "36 kB",
+  "zendeskgarden:max_size": "50 kB",
```

Comment:
This `max_size` value isn't correct here. Since we are using the `UMD` bundle to perform this check it's including multiple versions of several dependencies that wouldn't exist in a standard `commonjs` build.

One we transition to rollup we can use some more accurate bundle-analysis tooling.